DataScience/ProbabilisticParser/parser.py

```diff
@@ -50,12 +51,11 @@ def parse(raw_string):
     :return: a list of tokens and labels
     :rtype: list
     """
-
-    tokens = t.tokenize(raw_string)
+    tokens = tok.tokenize(raw_string)
```

Comment:
tokenize function does return an empty list. The following if is to force the parse function to return an empty list if there are no tokens to turn into features.